modules/lora_presets.py

```
@@ -0,0 +1,286 @@
+        [true, "None", 1.0]
+    ]
+}
+
+Each LoRA entry: [enabled (bool), filename (str), weight (float)]
+"""
+
+import os
+import json
+import re
+from datetime import datetime
+from typing import List, Tuple, Optional, Dict, Any
```

Ruff (0.14.5)

`typing.List` is deprecated, use `list` instead

(UP035)